src/code/z_view.c

```diff
@@ -368,7 +386,10 @@ s32 func_800AAA9C(View* view) {
     return 1;
 }
 
-s32 func_800AB0A8(View* view) {
+/**
+ * Subroutine of View_Apply
+ */
+s32 View_ApplyOrtho(View* view) {
```

Comment:
See name suggestion above. I'm also not entirely sure about "Apply". I also don't think how it's called is important, so I don't think "Subroutine of View_Apply" is needed

Reply:
OK, I've removed the comments. I've taken the prototypes of these out of the header which is a better way of saying they're internal helper functions, which I think they probably are.

bc8ad7c6d